portal-kernel/src/com/liferay/portal/kernel/servlet/MultiSessionErrors.java

```diff
@@ -92,7 +94,13 @@ private static boolean _isEmpty(HttpServletRequest httpServletRequest) {
 	}
 
 	private static Class<?>[] _hideDefaultErrorMessageClasses = {
-		NoSuchLayoutException.class
+		NoSuchLayoutException.class, UserPasswordException.class
 	};
 
+	static {
+		_hideDefaultErrorMessageClasses = ArrayUtil.append(
+			_hideDefaultErrorMessageClasses,
+			UserPasswordException.class.getDeclaredClasses());
```

Comment:
`getDeclaredClasses` <-- Nice! 

Can we add `UserPasswordException.class.getDeclaredClasses()` inline on L97?

Reply:
Yes, this is done, and I've made it static final.